package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "Brackets",
-    "version": "1.3.0-0",
-    "apiVersion": "1.3.0",
+    "version": "1.4.0-0",
```

Comment:
@walfgithub This bump up in version no is not required. Is this an automation error?